docs/recipes/auth/README.md

```diff
@@ -21,6 +21,7 @@ The server responds with different HTML for each of these pages and session stat
 ```sh
 # be in root lighthouse directory
 yarn # install global project deps
+npm run build-all # Build lighthouse
```

Comment:
```suggestion
yarn build-report
```

I think this is sufficient.

Reply:
Maybe, but you will not get me to spend time testing it. What I suggested worked for me.

Reply:
What would have worked for me: just hit accept, and I'll approve and merge this PR.